package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "bcryptjs": "^2.4.3",
+    "crypto": "^1.0.1",
```

Comment:
`crypto` is already a built-in module in Node.js, there is absolutely no need to install this module. Also it is already deprecated.

Reply:
Fixed! I have removed crypto